tardis/transport/montecarlo/packet_trackers.py

```diff
@@ -203,3 +204,42 @@ def track(self, r_packet):
         self.energy = r_packet.energy
         self.shell_id = r_packet.current_shell_id
         self.interaction_type = r_packet.last_interaction_type
+
+    # To make it compatible with RPacketTracker
+    def finalize_array(self):
+        pass
+
+
+@njit
+def generate_rpacket_tracker_list(no_of_packets, length):
+    """
+    Parameters
+    ----------
+    no_of_packets : The count of RPackets that are sent in the ejecta
+    length : initial length of the tracking array
+
+    Returns
+    -------
+    A list containing RPacketTracker for each RPacket
+    """
+    rpacket_trackers = List()
+    for i in range(no_of_packets):
+        rpacket_trackers.append(RPacketTracker(length))
+    return rpacket_trackers
+
+
+@njit
+def generate_rpacket_last_interaction_tracker_list(no_of_packets):
+    """
```

Comment:
@andrewfullard do these functions need tests?

Reply:
It might be useful, yes